src/containers/signupForm/index.tsx

```diff
@@ -0,0 +1,509 @@
+            </Form.Item>
+
+            <Form.Item
+              label="Last Name"
+              name="lastName"
+              className="inline-block-half"
+              rules={[
+                { required: true, message: 'Please input your last name' },
+              ]}
+            >
+              <Input placeholder="Last Name" />
+            </Form.Item>
+
+            <Form.Item
+              label="Pronouns"
+              name="pronouns"
+              rules={[{ message: 'Please select your pronouns' }]}
+            >
+              <Radio.Group>
+                <Radio className="radio-item" value="He/Him">
+                  He/Him
+                </Radio>
+                <Radio className="radio-item" value="She/Her">
+                  She/Her
+                </Radio>
+                <Radio className="radio-item" value="They/Them">
+                  They/Them
+                </Radio>
+              </Radio.Group>
+            </Form.Item>
+
+            <Form.Item
+              label="Email"
+              name="email"
+              rules={[
+                {
+                  required: true,
+                  type: 'email',
+                  message: 'Please input a valid email address',
+                },
+              ]}
+            >
+              <Input placeholder="Email" />
+            </Form.Item>
+
+            <Form.Item
+              label="Create Password"
+              name="password"
+              className="block-half stacked-inputs"
+              hasFeedback
+              rules={[
+                {
+                  required: true,
+                  pattern: new RegExp(/^(?=.*\d).{8,20}$/),
+                  message:
+                    'Password must be 8-20 characters, containing at least one digit',
+                },
+              ]}
+            >
+              <Input.Password placeholder="Password" />
+            </Form.Item>
+
+            <Form.Item
+              name="confirmPassword"
+              dependencies={['password']}
+              className="block-half"
+              hasFeedback
+              rules={[
+                { required: true, message: 'Passwords must match' },
+                ({ getFieldValue }) => ({
+                  validator(rule, value) {
+                    if (!value || getFieldValue('password') === value) {
+                      return Promise.resolve();
+                    }
+                    return Promise.reject('This password does not match!');
+                  },
+                }),
+              ]}
+            >
+              <Input.Password placeholder="Confirm Password" />
+            </Form.Item>
+
+            <Form.Item
+              label="Phone Number"
+              name="phoneNumber"
+              className="block-half"
+              rules={[
+                {
+                  required: true,
+                  pattern: new RegExp(/^[^0-9]*(?:(\d)[^0-9]*){10}$/),
```

Comment:
What does this enforce?

Reply:
I dont know, we really should test our regex whereever we use it, I didnt make this but im guessing this just makes sure its all digits